stdlib/stream.mli

```diff
@@ -67,10 +67,12 @@ val iter : ('a -> unit) -> 'a t -> unit
 
 val next : 'a t -> 'a
 (** Return the first element of the stream and remove it from the
-   stream. Raise {!Stream.Failure} if the stream is empty. *)
+   stream.
+   @raise Stream.Failure if the stream is empty. *)
 
 val empty : 'a t -> unit
-(** Return [()] if the stream is empty, else raise {!Stream.Failure}. *)
+(** Return [()]
```

Comment:
This function is also about expressly raising an exception - I think the description is better as it was without the `@raise` tag

Reply:
I will force push HEAD~